lessons/Early-Math/associative-property/langs/en-us/pages/categorize-functions.adoc

```diff
@@ -0,0 +1,59 @@
+= Categorizing Functions
```

Comment:
I LOVE this page!!!

One thing I would add (and you have the room for it) is to ask whether their answers are different for `@proglang` versions of +, -, *, and ÷ v. math